bin/node/node/src/service.rs

```diff
@@ -16,14 +16,24 @@
 
 //! Service and ServiceFactory implementation. Specialized wrapper over substrate service.
 
+// =====================================================================================
+// =====================================================================================
+// =====================================================================================
+// UPDATE GUIDE:
+// 1) replace everything with node-template/src/service.rs contents;
```

Comment:
```suggestion
// 1) replace everything with node-template/src/service.rs contents (found in main Substrate repo);
```

Reply:
I don't want a new person coming on-board and thinking that there's a missing`node-template` folder in this repo